src/Components/Login/Login.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {screen, fireEvent, render} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Login from './Login';
+import { BrowserRouter } from 'react-router-dom';
+
+describe('Login Component', () => {
+  it('should have two input fields and a button', () => {
+  render (<Login />);
+
+  const emailInput = screen.getByPlaceholderText('Username');
+  const passwordInput = screen.getByPlaceholderText('Password');
+  const submitButton = screen.getByRole('button', {name: 'Submit'});
+
+  expect(emailInput).toBeInTheDocument();
+  expect(passwordInput).toBeInTheDocument();
+  expect(submitButton).toBeInTheDocument();
+})
+
+  it('should check the input is clear after submit', () => {
```

Comment:
I would also make an assertion about the form component's inputs holding value after you change them. You can make another assertion about the value of the inputs, or look into the [byDisplayValue](https://testing-library.com/docs/dom-testing-library/api-queries#bydisplayvalue) query